packages/ui/index.tsx

```diff
@@ -125,8 +124,7 @@ export {
   RadioField,
 } from "./form/radio-area";
 
-export { default as MultiSelectCheckboxes } from "./components/form/checkbox/MultiSelectCheckboxes";
-export type { Option as MultiSelectCheckboxesOptionType } from "./components/form/checkbox/MultiSelectCheckboxes";
+export type { Option } from "./components/form/select/Select";
```

Comment:
this causes a type error 
<img width="1057" alt="Screenshot 2023-07-10 at 12 17 45" src="https://github.com/calcom/cal.com/assets/30310907/8564af55-de76-44d9-b0ba-40c1ef40883d">